grunt/helpers.js

```diff
@@ -34,6 +35,12 @@ module.exports = function(grunt) {
       for (let i = 0; i < count; i++) { grunt.log.writeln('- ' + excludes[i]); }
       grunt.log.writeln('');
     }
+    if (productionExcludes) {
+      let count = productionExcludes.length;
```

Comment:
minor, but `const` would be more appropriate here since the value of `count` doesn't get changed